docs/guides/chip_tool_guide.md

```diff
@@ -730,40 +731,40 @@ read or write. Attributes from that cluster can also be subscribed by the
 controller, which means that the `chip-tool` will receive notifications, for
 instance: in case the attribute value is changed or a particular event happens.
 
-#### Examples:
+#### Examples
+
+This is a list of `chip-tool` exemplary commands dedicated to particular use cases.
+
+##### Writing ACL (Access Control List) to the `accesscontrol` cluster
 
-##### Writing ACL (Access Control List) to the `accesscontrol` cluster:
+More information regarding ACL can be found in the [Access Control Guide](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/access-control-guide.md).
```

Comment:
What about moving this sentence below an example? You wrote "More information..." but there is no previous information on this topic. Or maybe simply add one sentence more at the beginning.